src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -222,15 +222,24 @@ private void ThumbResizeThumbOnDragDeltaHandler(object sender, DragDeltaEventArg
 
         private void CompositionTargetRenderingHandler(object sender, EventArgs e)
         {
-            //Do not call the clip plane update on the render loop if the camera is unchanged or
-            //the user is manipulating the view with mouse.  Do run when queued by runUpdateClipPlane bool 
-            if (runUpdateClipPlane || (!View.Camera.Position.Equals(prevCamera) && !View.IsMouseCaptured) )
+            // https://github.com/DynamoDS/Dynamo/issues/7295
+            // This should not crash Dynamo when View is null
+            try
             {
-                ViewModel.UpdateNearClipPlane();
-                runUpdateClipPlane = false;
+                //Do not call the clip plane update on the render loop if the camera is unchanged or
+                //the user is manipulating the view with mouse.  Do run when queued by runUpdateClipPlane bool 
+                if (runUpdateClipPlane || (!View.Camera.Position.Equals(prevCamera) && !View.IsMouseCaptured))
+                {
+                    ViewModel.UpdateNearClipPlane();
+                    runUpdateClipPlane = false;
+                }
+                ViewModel.ComputeFrameUpdate();
+                prevCamera = View.Camera.Position;
             }
-            ViewModel.ComputeFrameUpdate();
-            prevCamera = View.Camera.Position;          
+            catch(Exception ex)
+            {
+                Console.WriteLine(ex.ToString());
```

Comment:
possible to use the logger here?

Reply:
@mjkkirschner I tried but does not seem possible since we will need to pass a Logger instance from DynamoModel all the way down to node and keep it. We should make Logger a singleton in the future so that node level message could be written.

Reply:
This is not a node class - you can get the logger instance from the `HelixWatch3DViewModel` which is a field in this class I think.

Reply:
@mjkkirschner Thanks, checkout the change I made. I did not expect NodeViewModel to keep a reference of WorkspaceViewModel..